src/sentry/api/event_search.py

```diff
@@ -1561,6 +1566,13 @@ def normalize(self, value, params):
         raise InvalidFunctionArgument(u"{} is not a valid string array column".format(value))
 
 
+class StringColumn(FunctionArg):
+    def normalize(self, value, params):
+        if value in ["release"]:
```

Comment:
It might be more future proof and easier to read the function list if the allowed fields was a constructor argument. Like `StringColumn('that', allowed=['release'])`